resources/views/pages-legacy/userInfo.blade.php

```diff
@@ -176,7 +176,7 @@
     if ($userWallActive) {
         // passing 'null' for $user disables the ability to add comments
         RenderCommentsComponent(
-            !isUserBlocking($userPage, $user) ? $user : null,
+            !$userPageModel->isBlocking(request()->user()) ? $user : null,
```

Comment:
```
App\Models\User::isBlocking(): Argument #1 ($user) must be of type App\Models\User, null given, called in /var/www/html/storage/framework/views/4d718b16de0564107bea562ee3b2c8ee.php on line 218
```
if not logged in.

Reply:
Thanks, updated latest to also include a call to `request()->user()` in the conditional.

The inner call to `request()->user()` (which is cached) is still there. I considered lifting all this up to another variable, but wasn't sure what to name the variable that wouldn't make it ambiguous with all the other stuff happening around the user page.